changes/zh-cn/develop.md

```diff
@@ -3,6 +3,7 @@
 <!-- 请根据PR的类型添加 `变更记录` 到以下对应位置(feature/bugfix/optimize/test) 下 -->
 
 ### feature：
+  - [[#4676](https://github.com/seata/seata/pull/4676)] 新增nacos注册中心公网地址服务发现机制
```

Comment:
The commit log must have an exact meaning. What is the public network for open source users?